src/scss/design-system-interface.scss

```diff
@@ -156,13 +156,17 @@ $c-ame-orange: #F26531;
 
   &__sub {
     font-family: $primary-font;
-    font-size: 16px;
+    font-size: 1rem;
+    font-weight: 800;
+    text-transform: uppercase;
     display: block;
+    margin-bottom: 1rem;
   }
 
   &__text {
     font-family: $primary-font;
-    font-size: 36px;
+    font-size: 48px;
```

Comment:
@zastrow Do we want to have font-size variables set?

Reply:
It could be useful. This file is specifically dealing with the interface for the design system and isn’t the styles for the actual site, so really I wouldn’t worry too much about it when it comes to this file in particular.